views/components/info/control.es

```diff
@@ -170,6 +237,7 @@ const PoiControl = connect((state, props) => ({
   render() {
     return (
       <div className='poi-control-container'>
+        <script>{this.handleTouchbar(this.props.tbtriggered)}</script>
```

Comment:
Seems you should call it in a proper updating cycle, or call it before `render`'s return

see https://facebook.github.io/react/docs/react-component.html#the-component-lifecycle